### PR TITLE
FHIR Adapters

### PR DESCRIPTION
**Pull request by jsmith2**
MPI, registry, and repository FHIR adapters with conversions from FHIR resources to responses.

### CODE REVIEW

File: FHIRAdapters/src/main/java/org/hl7/fhir/client/ClientUtils.java [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+			value = IOUtils.toString(instream, "UTF-8");
+			System.out.println(value);
+			
+		} catch(IOException ioe) {
+			//Do nothing
+		}
+		return value;
+	}
+	
+  public static AtomFeed issuePostFeedRequest(URI resourceUri, Map<String, String> parameters, String resourceName, Resource resource, String feedFormat) throws Exception {
```
Comment (nareshcgi) [inline FHIRAdapters/src/main/java/org/hl7/fhir/client/ClientUtils.java]: Class needs formatting.

**Comment by nareshcgi** [comment]
Please add class level comments and also javadoc comments.

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without comment.

**Comment by TabassumJafri** [comment]
Reviewed the PR and it looks good

**Comment by alameluchidambaram** [comment]
Reviewed PR and looks good

**Comment by sailajaa** [comment]
I have reviewed the PR and good with it.

**Pull request merged**